SPECS/openssl/openssl.spec

```diff
@@ -257,6 +256,9 @@ rm -f $RPM_BUILD_ROOT%{_sysconfdir}/pki/tls/ct_log_list.cnf.dist
 rm -rf $RPM_BUILD_ROOT
 
 %changelog
+* Tue Nov 10 2020 Johnson George <johgeorg@microsoft.com> 1.1.1g-6
```

Comment:
Please update the manifest files (look for "openssl" inside .txt files in the project and you'll immediately find them.